use-timescale/ha-replicas/read-scaling.md

```diff
@@ -1,78 +1,137 @@
-with read-only permissions cannot access the primary data instance directly.
+A $READ_REPLICA set in $CLOUD_LONG is a group of one or more $READ_REPLICA nodes that are accessed through the same endpoint. You query each set as a single replica. $CLOUD_LONG balances the load between the nodes in the set for you.
 
-## Create a read replica
+![Read scaling in Timescale](https://assets.timescale.com/docs/images/read-scaling-timescale.png)
 
-Read replicas can be short-lived and deleted when the analysis is complete, or long-running to power a
-business intelligence (BI) tool. To create a secure read replica for your read-intensive apps: 
+You can create as many $READ_REPLICA sets as you need. For security and resource isolation, each $READ_REPLICA set has unique connection details.
+
+You use $READ_REPLICA sets for horizontal **read** scaling. To limit data loss for your $SERVICE_LONGs, use [$HA_REPLICAs][ha].
+
+## Prerequisites
+
+To follow this procedure:
+
+- Create a target $SERVICE_LONG.
+- Create a [read-only user][read-only-role] on the primary data instance. 
+
+  A user with read-only permissions cannot make changes in the primary database. This user is propagated to the $READ_REPLICA set when you create it.
+
+## Create a $READ_REPLICA set
+
+To create a secure $READ_REPLICA set for your read-intensive apps: 
 
 <Procedure>
 
-1. Best practice is to create a [read-only role][read-only-role] for the person using the replica.
+1. **In [$CONSOLE][timescale-console-services], select your target $SERVICE_SHORT**
+
+1. **Click `Operations` > `Read scaling` > `Add a read replica set`**
+
+1. **Configure your replica set** 
+
+    Configure the number of nodes, compute size, connection pooling, and the name for your replica, then click `Create read replica set`.
 
-   You create the read-only user on the primary data instance. This user is propagated to the read
-   replica when you create it.
-1. In [Timescale Console][timescale-console-services], select the service to replicate.
-1. Click `Operations`, then click `Read replicas`.
-1. Click `Add read replica`, then select the configuration you want and click `Add read replica`.
-1. Note the connection information for the read replica. 
+   ![Create a read replica set in Timescale Console](https://assets.timescale.com/docs/images/create-read-replica-set-timescale-console.png)
 
-    The connection string for each read replica is unique, and different to one you use for the primary data instance. 
+1. **Save the connection information**
+
+   The username and password of a read replica set are the same as the primary $SERVICE_SHORT. They cannot be changed independently. 
+
+   The connection information for each $READ_REPLICA set is unique. If you add or remove nodes from an existing set, the connection information of that set changes. To find the connection information for an existing $READ_REPLICA set: 
```

Comment:
```suggestion
   The connection information for each $READ_REPLICA set is unique. You can you add or remove nodes from an existing set and the connection information of that set will remain the same. To find the connection information for an existing $READ_REPLICA set: 
```

Reply:
@ramonguiu just checking: this was originally in the PRD.  I have tested this on dev and it seems the connection string did change when changing the number of nodes?